README.md

```diff
@@ -28,22 +53,53 @@ Backpointers
 This proposal is based on the LDML spec, List Patterns:
 
 
-### Prior Art
+## Proposal
 
 
-### Usage
+### Spec
 
-```javascript
-let o = new Intl.ListFormat("en", {
-    style: "regular" // default style
-});
-console.log(o.format(['foo', 'bar', 'baz']); // "foo, bar, and baz"
-```
+You can view the [spec text](spec/listformat.html).
+
+
+### API
+
+#### `Intl.ListFormat([locales[, options]])`
+
+The `Intl.ListFormat` object is a constructor for objects that enable language-sensitive list formatting.
+
+#### `locales`
+
+Optional. A string with a BCP 47 language tag, or an array of such strings. For the general form and interpretation of the `locales` argument, see the [`Intl` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation).
+
+#### `options`
+
+Optional. An object with some or all of the following properties:
+
+##### `localeMatcher`
+
+The locale matching algorithm to use. Possible values are `"lookup"` and `"best fit"`; the default is `"best fit"`. For information about this option, see the  [`Intl` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation).
```

Comment:
This option is semi-useless; I'd put it at the bottom of the list.

Reply:
Probably will use alphabetical order, to be more consistent